pkg/proto/query_coord.proto

```diff
@@ -1012,6 +1014,43 @@ message ValidateAnalyzerRequest{
     repeated AnalyzerInfo analyzer_infos = 2;
 }
 
+// HighlightTask fetch highlight for all queries at one field
+// len(target_texts) == len(topks) == nq
+// len(texts) == sum(topks)
+message HighlightTask{
+    string field_name = 1;
+    int64 field_id = 2;
+    repeated string texts = 3;
```

Comment:
a single HighlightTask include all NQs? what if some query did not get sufficient k results?

Reply:
> a single HighlightTask include all NQs? what if some query did not get sufficient k results?

Tops was true num of result for each nq.